src/Tests/FlatSharpEndToEndTests/FlatSharpEndToEndTests.csproj

```
@@ -56,6 +56,51 @@
 
   <ItemGroup>
     <FlatSharpSchema Include="**\*.fbs" />
+    <FlatSharpSchema Remove="ToStringMethods\ToStringMethods.fbs" />
   </ItemGroup>
+
+  <Target Name="FlatSharpFbsCompileToString" BeforeTargets="ResolveAssemblyReferences">
+       <PropertyGroup>
+               <CompilerVersion>net8.0</CompilerVersion>
+       </PropertyGroup>
+
+       <!-- try .net6.0. -->
+       <PropertyGroup Condition=" $([System.Text.RegularExpressions.Regex]::IsMatch($(StdOut), '6\.0\.\d+')) ">
+               <CompilerVersion>net6.0</CompilerVersion>
+       </PropertyGroup>
+
+       <!-- try .net7.0. -->
+       <PropertyGroup Condition=" $([System.Text.RegularExpressions.Regex]::IsMatch($(StdOut), '7\.0\.\d+')) ">
+               <CompilerVersion>net7.0</CompilerVersion>
+       </PropertyGroup>
+
+       <!-- try .net8.0. -->
+       <PropertyGroup Condition=" $([System.Text.RegularExpressions.Regex]::IsMatch($(StdOut), '8\.0\.\d+')) ">
+               <CompilerVersion>net8.0</CompilerVersion>
+       </PropertyGroup>
+
+       <PropertyGroup>
+               <FlatSharpOutput>$(IntermediateOutputPath)tostring</FlatSharpOutput>
+               <FlatSharpOutput Condition=" '$(FlatSharpMutationTestingMode)' == 'true' ">$(MSBuildProjectDirectory)/</FlatSharpOutput>
```

```suggestion
               <FlatSharpOutput>$(IntermediateOutputPath)</FlatSharpOutput>
               <FlatSharpOutput Condition=" '$(FlatSharpMutationTestingMode)' == 'true' ">$(MSBuildProjectDirectory)/</FlatSharpOutput>
               <FlatSharpOutput>$(FlatSharpOutput)ToStringMethods</FlatSharpOutput>
```
Should handle the mutation testing too, I think